src/sentry/migrations/0212_fix_artifact_counts.py

```diff
@@ -0,0 +1,59 @@
+# Generated by Django 1.11.29 on 2021-06-21 07:28
+
+from django.db import migrations
+from django.db.models import Q
+
+from sentry.models.releasefile import ARTIFACT_INDEX_FILENAME
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def uncount_internal_releasefiles(apps, schema_editor):
+    # artifact_count=None implicitly means "count as 1".
+    # Internal release files (artifact index, release archives) created *before*
+    # 0211_add_artifact_count must not be counted as 1 but rather as 0, because their
+    # individual files were saved with a count of None as well.
+    ReleaseFile = apps.get_model("sentry", "ReleaseFile")
+    db_alias = schema_editor.connection.alias
+    qs = ReleaseFile.objects.using(db_alias).filter(
+        Q(name=ARTIFACT_INDEX_FILENAME)
+        | (Q(name__startswith="release-artifacts") & Q(name__endswith=".zip")),
```

Comment:
`startswith` and `endswith` use `LIKE` in sql, which can't actually make use of a postgres index, even if we had one on `name`. We'd probably have to add a functional index specifically for this backfill so that it will be performant.